programs/perpetuals/src/state/user_staking.rs

```diff
@@ -0,0 +1,265 @@
+//! Stake state and routines
```

Comment:
Might want to remove this, hard to keep this kind of comment relevant, better use gitbook